src/components/Integrations/index.js

```diff
@@ -42,7 +42,7 @@ export const IntegrationsSection = InjectAppServices(
   ({ dependencies: { controlPanelService, dopplerUserApiClient } }) => {
     const intl = useIntl();
     const _ = useCallback((id, values) => intl.formatMessage({ id }, values), [intl]);
-    const [{ integrationSection, loading }, dispatch] = useReducer(
+    const [{ integrationSection, loading, loadingStatus }, dispatch] = useReducer(
```

Comment:
I do not understand the name `loadingStatus`, we already have a `loading` _status_, do not we?

Is it a kind of error?

Reply:
hmm yes, we do have a loading already but that is beeing used for the page loading. 
This new loading I'm using it for the native integrations only, since there is a delay between the rendering and the boxes re-ordering.
Maybe there is a better way to handle it? or the name might be confusing?

Reply:
Maybe renaming the properties as `loadingPage` and `loadingNativeIntegrations`?

Reply:
I like it! will do...